cypher/refcard-tests/src/test/scala/org/neo4j/cypher/docgen/refcard/IndexTest.scala

```diff
@@ -185,9 +185,32 @@ MATCH (n:Person) WHERE n.name = $$value
 RETURN n
 ###
 
-An index can be automatically used for the equality comparison.
+An BTREE index can be automatically used for the equality comparison.
 Note that for example `toLower(n.name) = $$value` will not use an index.
 
+###assertion=match
+//
+
+MATCH (n:Person) WHERE n.name = "Alice"
+
+RETURN n
+###
+
+An TEXT index can be automatically used for the equality comparison when comparing to a string.
```

Comment:
```suggestion
A TEXT index can be automatically used for the equality comparison when comparing to a string.
```

Reply:
Should index types have code formatting?
```suggestion
A `TEXT` index can be automatically used for the equality comparison when comparing to a string.
```